taxonomy-editor-frontend/src/pages/editentry/index.tsx

```diff
@@ -119,4 +126,30 @@ const EditEntry = ({ addNavLinks }) => {
   );
 };
 
-export default EditEntry;
+type EditEntryWrapperProps = {
+  addNavLinks: ({
+    branchName,
+    taxonomyName,
+  }: {
+    branchName: string;
+    taxonomyName: string;
+  }) => void;
+};
+
+const EditEntryWrapper = ({ addNavLinks }: EditEntryWrapperProps) => {
+  const { taxonomyName, branchName, id } = useParams();
+
+  if (!taxonomyName || !branchName || !id)
+    return <div>Ooops, something went wrong! Please try again later.</div>;
```

Comment:
Maybe this could be added inside a Typography, since the text size would be uniform.

Reply:
done, added `Typography`